doc/Programming-guide/Key-platform-features/Devices-provisioning-and-registration/index.md

```diff
@@ -6,5 +6,131 @@ nav: /:path/Programming-guide/Key-platform-features/Devices-provisioning-and-reg
 sort_idx: 10
 ---
 
-Devices provisioning and registration
+* TOC
+{:toc}
 
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+Kaa establishes a mechanism to uniquely identify each endpoint and the associated tokens and credentials issued to that endpoint.
```

Comment:
In the first section you should talk about overall credentials service functionality (provision, revoke, mark in use).